courses/templates/course_detail.html

```diff
@@ -79,20 +79,36 @@ <h2>{{ course.subhead }}</h2>
             </ul>
         </div>
     </div>
- {% if course.outcomes %}
-     <div class="learning-outcomes-block">
-         <div class="container">
-             <div class="learning-header">
-                 <h1>{{ course.outcomes.heading | safe }}</h1>
-                 <h3>{{ course.outcomes.sub_heading | safe }}</h3>
+    {% if course.outcomes %}
+        <div class="learning-outcomes-block">
+             <div class="container">
+                 <div class="learning-header">
+                     <h1>{{ course.outcomes.heading | safe }}</h1>
+                     <h2>{{ course.outcomes.sub_heading | safe }}</h2>
+                 </div>
+                 <ul class="learning-outcomes-list">
+                     {% for outcome in course.outcomes.outcome_items %}
+                         <li>{{ outcome }}</li>
+                     {% endfor %}
+                 </ul>
              </div>
-             <ul class="learning-outcomes-list">
-                 {% for outcome in course.outcomes.outcome_items %}
-                     <li>{{ outcome }}</li>
-                 {% endfor %}
-             </ul>
-         </div>
-     </div>
- {% endif %}
+        </div>
+    {% endif %}
+    {% if course.techniques %}
+        <div class="container learning-techniques-block">
+            <h1>How You’ll Learn</h1>
+            <ul class="learning-techniques-list">
+                {% for technique in course.techniques.technique_items %}
+                    <li>
+                        <div class="img-holder">
+                            {% image technique.value.image height-110 format-png %}
```

Comment:
I think we need to change this to `fill-110x110` or some similar dimensions instead of `height-110`. If a user uploads an image with a width that is reasonably greater than its height, it will break the grid look.

Reply:
with height-110, width will be set accordingly auto. 
I tried fill-110x110 before and now too but Image is still breaking with that.

Reply:
Can you show me how `fill-110x110` is breaking? Here's a wide image in mobile view:

<img width="361" alt="ss 2019-05-03 at 11 47 16 " src="https://user-images.githubusercontent.com/14932219/57149460-c429ac00-6d99-11e9-92dd-67f890774fc6.png">

It's fine to suggest to the content managers that they should just not upload long images, but we're not validating the dimensions for the images they upload, so there's always the possibility for human error here. If you can show me how `fill-110x110` is breaking the UI, I guess we can keep it as-is

Reply:
<img width="355" alt="Screen Shot 2019-05-03 at 9 58 03 PM" src="https://user-images.githubusercontent.com/7334669/57152983-8b67f200-6dee-11e9-8110-855f4101903f.png">